packages/kbn-apm-synthtrace/src/lib/apm/span.ts

```diff
@@ -83,3 +96,77 @@ export function dbExitSpan({ spanName, spanSubType }: { spanName: string; spanSu
     'span.destination.service.type': spanType,
   };
 }
+
+export function elasticsearchSpan(spanName: string, statement?: string) {
+  const spanType = 'db';
+  const spanSubType = 'elasticsearch';
+
+  return {
+    spanName,
+    spanType,
+    spanSubType,
+
+    ...(statement
+      ? {
+          'span.db.statement': statement,
+          'span.db.type': 'elasticsearch',
+          'span.action': 'query',
+        }
+      : {}),
+
+    'service.target.type': spanSubType,
+    'destination.address': 'qwerty.us-west2.gcp.elastic-cloud.com',
+    'destination.port': 443,
+    'span.destination.service.name': spanSubType,
+    'span.destination.service.type': spanType,
+    'span.destination.service.resource': spanSubType,
+  };
+}
+
+export function sqliteSpan(spanName: string, statement?: string) {
+  const spanType = 'db';
+  const spanSubType = 'sqlite';
+
+  return {
+    spanName,
+    spanType,
+    spanSubType,
+
+    ...(statement
+      ? {
+          'span.db.statement': statement,
+          'span.db.type': 'sql',
+          'span.action': 'query',
+        }
+      : {}),
+
+    // destination
+    'service.target.type': spanSubType,
+    'destination.address': 'qwerty.us-west2.gcp.elastic-cloud.com',
+    'destination.port': 443,
+    'span.destination.service.name': spanSubType,
```

Comment:
fyi: `span.destination.service.name` and `span.destination.service.type` are deprecated and will be removed in a future release. Do we have to include them?